scoring/update.html

```diff
@@ -1,8 +1,22 @@
 {% extends "_update.html" %}
 
+{% macro js_tokens_summary(where, corner) %}
+  <div class="js-summary" style="display: flex; flex-wrap: nowrap; justify-content: space-evenly;">
+    <div>
+      B: <strong id="{{ where }}-bronze-{{ corner }}">&hellip;</strong>
+    </div>
+    <div>
+      S: <strong id="{{ where }}-silver-{{ corner }}">&hellip;</strong>
+    </div>
+    <div>
+      G: <strong id="{{ where }}-gold-{{ corner }}">&hellip;</strong>
+    </div>
+  </div>
+{% endmacro %}
+
 {% macro input_robot_tokens_wider(x, y, corner) %}
 <foreignObject x="{{ x }}" y="{{ y }}" width="120" height="30">
-  <label for="robot_tokens_{{ corner }}">Robot Tokens</label>
+  <label for="robot_tokens_{{ corner }}">Robot {{ corner }}</label>
```

Comment:
is there an easy way to colour-code these?

Reply:
e214f18